git_acp/.env.example

```
@@ -0,0 +1,72 @@
+GIT_ACP_MAX_DIFF_PREVIEW_LINES=10
+
+# Max number of non-type groups to create when auto-grouping changes
+GIT_ACP_AUTO_GROUP_MAX_NON_TYPE_GROUPS=5
+
+# TERMINAL CONFIGURATION
+# Terminal Colors (supports rich color names)
+GIT_ACP_DEBUG_HEADER_COLOR=blue
+GIT_ACP_DEBUG_VALUE_COLOR=cyan
+GIT_ACP_SUCCESS_COLOR=green
+GIT_ACP_WARNING_COLOR=yellow
+GIT_ACP_STATUS_COLOR=bold green
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GIT_ACP_STATUS_COLOR key should go before the GIT_ACP_SUCCESS_COLOR key

(UnorderedKey)